### PR TITLE
Attendance Page

### PR DESCRIPTION
**Pull request by Apexal**
Screen Shot 2023-02-04 at 5 13 39 PM

### CODE REVIEW

File: portal/models.py [hunk 1]
```diff
@@ -410,6 +410,11 @@ def can_propose_project(
 
         return True, None
 
+    def get_expected_meetings(self, semester=Semester):
+        # Determine what kinds of meetings this student is expected to attend
+        meeting_types = [Meeting.LARGE_GROUP, Meeting.SMALL_GROUP, Meeting.WORKSHOP]
+        return Meeting.objects.filter(type__in=meeting_types, semester=semester)
```
Comment (bnidevs) [inline portal/models.py:416]: will this eventually vary based on student role?

File: portal/templates/portal/users/detail.html [hunk 2]
```diff
@@ -104,7 +105,12 @@ <h1 class="title is-1">{{ user.full_name }}</h1>
 
 <section class="section">
     <div class="container">
-        <p class="has-text-grey">Coming soon...</p>
+        {% if request.user.is_superuser or request.user == user and user.is_rpi%}
+        <div class="block">
+            <a href="{% url 'user_attendance' user.pk %}{% active_semester_query %}" class="button">Attendance</a>
+        </div>
+        {% endif %}
+        <p class="has-text-grey">More coming soon...</p>
```
Comment (bnidevs) [inline portal/templates/portal/users/detail.html:113]: what will go here?

**Review by bnidevs — APPROVED** [review]

File: portal/models.py (hunk 1 above)
Comment (Apexal) [reply portal/models.py:416]: exactly!

File: portal/templates/portal/users/detail.html (hunk 2 above)
Comment (Apexal) [reply portal/templates/portal/users/detail.html:113]: - [ ] a list of the semesters that the student have been enrolled in, and for each semester:
  - [ ] their team

**Pull request merged**